modules/layers/src/layers/editable-h3-layer.ts

```diff
@@ -0,0 +1,152 @@
+/* eslint-env browser */
+
+import { H3ClusterLayer } from '@deck.gl/geo-layers';
+import { ViewMode } from '@nebula.gl/edit-modes';
+import { polyfill } from 'h3-js';
+import EditableGeoJsonLayer from './editable-geojson-layer';
+import EditableLayer from './editable-layer';
+
+const DEFAULT_EDIT_MODE = ViewMode;
+const DEFAULT_H3_RESOLUTION = 9;
+
+const defaultProps = {
+  mode: DEFAULT_EDIT_MODE,
+
+  // EditableGeoJsonLayer
+  geoJsonData: {},
+  ...EditableGeoJsonLayer.defaultProps,
+
+  // h3 layer
+  h3Data: {},
```

Comment:
to make it consistent with other deck.gl layers, can we use the name `data` instead of `h3Data` for the prop name?

Reply:
also, default should be empty array `[]` instead of empty object `{}` I believe.

Reply:
Yup, makes sense.